docusaurus/docs/iOS/basics/integration.md

```diff
@@ -0,0 +1,184 @@
+  - `pod 'StreamChatUI', '~> 4.0.0'`
+- For the SwiftUI components (**StreamChatSwiftUI**, which depends on **StreamChat**) use:
+  - `pod 'StreamChatSwiftUI', '~> 4.0.0'`
+
+<details><summary>➤ Do you want to use <b>XCFrameworks</b>? (Click to read more)</summary>
+<p>
+
+:::caution
+Our XCFrameworks are built with **Swift 5.5**. In order to use them you need **Xcode 13** or above
+:::
+
+You can learn more about [our Module Stable XCFrameworks here](#xcframeworks)
+
+- For the LLC (**StreamChat**) use:
+  - `pod 'StreamChat-XCFramework', '~> 4.0.0'`
+- For the UIKit components (**StreamChatUI**, which depends on **StreamChat**) use:
+  - `pod 'StreamChatUI-XCFramework', '~> 4.0.0'`
+
+</p>
+</details>
+
+Example: If you want to use the UIKit components, it should look similar the snippet below.
+
+```ruby
+target 'MyProject' do
+  use_frameworks!
+
+  # Pods for MyProject
+  pod 'StreamChatUI', '~> 4.0.0'
+end
+```
+
+StreamChatUI and StreamChatSwiftUI pod will automatically include the StreamChat dependency. If you want just the StreamChat dependency, without the UI components, use StreamChat as shown above.
+
+Now that we’ve modified our Podfile, let’s go ahead and install the project dependencies via the terminal with one simple command:
+
+```bash
+pod install --repo-update
+```
+
+The above command will generate the **MyProject.xcworkspace** file automatically.
+
+With our workspace now containing our Pods project with dependencies, as well as our original project, let’s go ahead and move over to Xcode to complete the process.
+
+_More information about CocoaPods [can be found here](https://cocoapods.org/)_
+
+### Carthage
+
+In your project's `Cartfile`, add one of these options
+
+- For the LLC (**StreamChat**) use:
+  - `github "getstream/stream-chat-swift" ~> 4.0.0`
+- For the UIKit components (**StreamChatUI**, which depends on **StreamChat**) use:
+  - `github "getstream/stream-chat-swift" ~> 4.0.0`
+
+:::note
+Our SwiftUI components library is not yet available using Carthage, please use Swift Package Manager or CocoaPods.
+:::
+
+
+<details><summary>➤ Do you want to use <b>XCFrameworks</b>? (Click to read more)</summary>
+<p>
+
+:::caution
+Our XCFrameworks are built with **Swift 5.5**. In order to use them you need **Xcode 13** or above
+:::
+
+You can learn more about [our Module Stable XCFrameworks here](#xcframeworks)
+
+- For the LLC (**StreamChat**) use:
+  - `binary "https://raw.githubusercontent.com/GetStream/stream-chat-swift/main/StreamChatArtifacts.json" ~> 4.0`
+- For the UIKit components (**StreamChatUI**, which depends on **StreamChat**) use:
+  - `binary "https://raw.githubusercontent.com/GetStream/stream-chat-swift/main/StreamChatArtifacts.json" ~> 4.0`
+
+</p>
+</details>
+
+Now that we’ve modified our Cartfile, let’s go ahead and install the project dependencies via the terminal with one simple command:
+
+```bash
+carthage update --use-xcframeworks
```

Comment:
```suggestion
carthage update --use-xcframeworks --platform iOS
```

Reply:
Not really, we are supporting macOS for StreamChat 🤷‍♂️. Also, whenever we add XCFrameworks as default option, passing a platform does not automatically pick them

Reply:
Interesting, I was encountering the issues when the flag wasn't specified :)